client/my-sites/checkout/checkout/credit-card-payment-box.jsx

```diff
@@ -159,15 +162,6 @@ export class CreditCardPaymentBox extends React.Component {
 		);
 	};
 
-	paymentBoxActions = () => {
-		let content = this.paymentButtons();
-		if ( this.props.transactionStep && this.submitting( this.props.transactionStep ) ) {
```

Comment:
I was wondering why you removed the check on whether `this.props.transactionStep` is set, but then realized it's a required prop - nice catch! :)